keylime/ima/ima.py

```diff
@@ -610,6 +613,30 @@ def canonicalize_allowlist(alist_bytes, checksum=""):
     return alist
 
 
+def ima_policy_db_contents(ima_policy_name: str, ima_policy: str, tpm_policy: str = "") -> dict:
+    """Assembles an ima policy dictionary to be written on the database"""
+    ima_policy_db_format = {}
+    ima_policy_db_format["name"] = ima_policy_name
+    # TODO: This was required to ensure e2e CI tests pass
+    if ima_policy == "{}":
+        ima_policy_db_format["ima_policy"] = None
+    else:
+        ima_policy_db_format["ima_policy"] = ima_policy
+    ima_policy = json.loads(ima_policy)
```

Comment:
shouldn't this be ima_policy_json ?

Reply:
Ah I think that @stefanberger has a point:

```
ima_policy_db_format = {"name": ima_policy_name, "ima_policy": None}
ima_policy = json.loads(ima_policy_json)
if ima_policy:
    ima_policy_db_format["ima_policy"] = ima_policy_json
```

That should be more correct, as there are multiple representation of an empty policy when you compare the JSON string, but there is only one (not the `len`) when you compare the `dict`